_maps/shuttles/construction_mining.dmm

```diff
@@ -0,0 +1,145 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 9
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"b" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"c" = (
+/obj/machinery/computer/shuttle{
+	possible_destinations = "construction_mining_custom";
+	shuttleId = "construction_mining"
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"d" = (
+/obj/structure/table,
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"e" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 5
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"f" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 8
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"g" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 4
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"h" = (
+/obj/docking_port/mobile{
+	dir = 8;
+	dwidth = 3;
+	height = 5;
+	id = "construction_mining";
+	name = "spare mining shuttle";
+	port_direction = 4;
+	width = 7
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 4
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"i" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 10
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"j" = (
+/obj/structure/shuttle/engine/heater,
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"k" = (
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"l" = (
+/obj/structure/shuttle/engine/propulsion/burst,
+/obj/effect/turf_decal/stripes/line,
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"z" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 6
+	},
+/turf/open/floor/plating/airless,
+/area/shuttle/mining)
+"J" = (
+/obj/machinery/computer/camera_advanced/shuttle_docker{
+	jumpto_ports = list(mining_home = 1, mining_away = 1, landing_zone_dock = 1, mining_public = 1, construction_mining_custom = 1);
+	shuttleId = "construction_mining";
+	shuttlePortId = "construction_mining_custom"
```

Comment:
I'm not allowing player-built shuttles with custom docking locations. This `construction_mining_custom` looks suspiciously like a player-defined docking location.

You're advertising this as a buyable spare mining shuttle, so it should be limited to all the same docking ports as the existing mining shuttle.

If this means you need to add a secondary docking areas to allow this to function properly, then that's fine.

Also this should not be a var edit hidden in a mapping file.